system/Helpers/url_helper.php

```diff
@@ -634,3 +634,46 @@ function mb_url_title(string $str, string $separator = '-', bool $lowercase = fa
 }
 
 //--------------------------------------------------------------------
+
+if (! function_exists('url_to'))
+{
+	/**
+	 * Get the full, absolute URL to a controller method
+	 * (with additional arguments)
+	 *
+	 * @param string $controller
+	 * @param mixed  ...$args
+	 *
+	 * @throws \CodeIgniter\Router\Exceptions\RouterException
+	 *
+	 * @return string
+	 */
+	function url_to(string $controller, ...$args): string
+	{
+		if (strpos($controller, '::') !== false)
+		{
+			$class = explode('::', $controller)[0];
+
+			if (! class_exists($class))
+			{
+				$controller = service('routes')->getDefaultNamespace() . $controller;
+			}
+		}
```

Comment:
We should run this code only if `route_to()` fail. In other case, it's just a waste of time.

Reply:
@ImMaax Looks like we're almost there! Could you move this around and run tests again?

Reply:
@MGatner sure, sure. I'm currently busy, but will work on this later today!